test/system/notifier-rewards.test.js

```diff
@@ -111,7 +105,7 @@ describeFn("System -- notifier rewards", () => {
       await depositStub.notifyUndercollateralizedLiquidation()
     })
 
-    context("when both reward amount and percentage are zero", () => {
+    context("when reward amount is zero", () => {
       beforeEach(async () => {
         await riskManagerV1
           .connect(notifier)
```

Comment:
`context("when reward amount is set but percentage is zero", () => {` at line 122 needs to be renamed.

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/146/commits/b57fb989c20a943dbe6d224196f2bbc50360fa9b